supriya/tools/ugentools/Duty.py

```diff
@@ -1,9 +1,9 @@
 # -*- encoding: utf-8 -*-
 from supriya.tools.ugentools.UGen import UGen
-
+from supriya import *
 
 class Duty(UGen):
-    r'''
+    r'''A value is demanded of each UGen in the list and output according to a stream of duration values.
 
     ::
```

Comment:
For the `Duty` example, build it off of an example from the SC documentation to make the usage more obvious:

```
duty = ugentools.Duty.kr(
    done_action=0,
    duration=ugentools.Drand(
        sequence=[0.01, 0.2, 0.4],
        repeats=2,
        ),
    reset=0,
    level=ugentools.Dseq(
        sequence=[204, 400, 201, 502, 300, 200],
        repeats=2,
        ),
    )
```